app/javascript/pages/account/account-settings.tsx

```diff
@@ -35,27 +35,27 @@ import { AxiosError } from "axios"
 import { ErrorSummaryBanner } from "./components/ErrorSummaryBanner"
 import { ExpandedAccountAxiosError, getErrorMessage } from "./components/util"
 
-const SavedBanner = () => {
-  return (
-    <Alert fullwidth className="account-settings-banner">
-      {t("accountSettings.accountChangesSaved")}
-    </Alert>
-  )
-}
-
-const UpdateBanner = () => {
-  return (
-    <Alert fullwidth className="account-settings-banner">
-      {t("accountSettings.update")}
-    </Alert>
-  )
-}
-
-const EmailBanner = () => {
+const Banner = ({
```

Comment:
future refactor suggestion: We should consider pulling this into its own component file for easier reusability